lms/djangoapps/certificates/views.py

```diff
@@ -591,6 +597,16 @@ def render_html_view(request, user_id, course_id):
     # Append/Override the existing view context values with any course-specific static values from Advanced Settings
     context.update(course.cert_html_view_overrides)
 
+    # track certificate evidence_visited event for analytics when certificate_user and accessing_user are different
+    if request.user and request.user.id != user.id:
+        emit_certificate_event('evidence_visited', user, course_id, course, {
```

Comment:
typically we put `request.user` into the "context.user_id" field in the event. This indicates which user made the request.

If the request pertains to another "target" user (as it does in this case), we put that second "target" user's ID into the body (like you do here).

I'm thinking, though, that you should be passing in `request.user` instead of `user` as the second parameter of this function call.